packages/next/src/generators/library/library.spec.ts

```diff
@@ -152,14 +152,14 @@ describe('next library', () => {
     expect(appTree.read('my-lib/src/index.ts', 'utf-8')).toContain(
       'React client components'
     );
-    expect(appTree.read('my-lib/src/server.ts', 'utf-8')).toContain(
+    expect(appTree.read('my-lib/src/server.tsx', 'utf-8')).toContain(
```

Comment:
Had to generate with a default component, otherwise the Standalone app complains about no exports being found from module.